Jenkinsfile

```diff
@@ -0,0 +1,23 @@
+pipeline {
+    agent any
+    stages {
+        stage('Build docs') {
+            steps {
+                sh 'hatch run docs:build'
+            }
+        }
+    }
+    post {
+      success {
```

Comment:
When the stages and steps gets more complicated. It might make more sense to publish within the step. And not only when the whole project passes. But fine for now.

Reply:
Noted